core/services/ocr2/plugins/ocr2keeper/evm21/upkeep_info_test.go

```diff
@@ -10,6 +11,31 @@ import (
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
 )
 
+func TestTriggerID(t *testing.T) {
+	// hex string: 0xb68ad824000000000000000000000000d7e8be1f3e60f7e4c1dc4445cd4f67f2
+	bigIntStr := "82566255084862886500628610724995377215109748679571001950554849251333329872882"
+	// Convert the string to a big.Int
+	var bigIntValue big.Int
```

Comment:
nitpick - maybe call this `upkeepID` so that we know how it's being used in the test?